### PR TITLE
[NUI] Fix extents issue and unified style format

### PR DESCRIPTION
**Pull request by elishateng**
### Description of Change ###

1. Fix extents issue
2. Unified style format

### API Changes ###

 - ACR:

### CODE REVIEW

File: src/Tizen.NUI.Components/Style/ButtonStyle.cs [hunk 1]
```diff
@@ -89,6 +82,7 @@ public class ButtonStyle : ControlStyle
         public static readonly BindableProperty IconPaddingProperty = BindableProperty.Create(nameof(IconPadding), typeof(Extents), typeof(ButtonStyle), null, propertyChanged: (bindable, oldValue, newValue) =>
         {
             var buttonStyle = (ButtonStyle)bindable;
+            if (null == buttonStyle.iconPadding) buttonStyle.iconPadding = new Extents(buttonStyle.OnIconPaddingChanged, 0, 0, 0, 0);
```
Comment (rabbitfor) [inline src/Tizen.NUI.Components/Style/ButtonStyle.cs]: Could you help me to understand this code?  
After setting `buttonStyle.iconPadding` to new Extents, the next line is re-setting it to `(Extents)newValue`.

File: src/Tizen.NUI.Components/Style/SliderStyle.cs [hunk 2]
```diff
@@ -228,25 +230,13 @@ public SliderStyle(SliderStyle style) : base(style)
...
-                {
-                    _trackPadding = new Extents((ushort start, ushort end, ushort top, ushort bottom)=>
-                                        {
-                                            Extents extents = new Extents(start, end, top, bottom);
-                                            _trackPadding.CopyFrom(extents);
-                                        }, 0, 0, 0, 0);
-                }
-                return _trackPadding;
+                Extents tmp = (Extents)GetValue(TrackPaddingProperty);
+                return (null == tmp) ? tmp : trackPadding = new Extents(OnTrackPaddingChanged, 0, 0, 0, 0);
```
Comment (rabbitfor) [inline src/Tizen.NUI.Components/Style/SliderStyle.cs]: (null != tmp) ?

File: src/Tizen.NUI/src/public/BaseComponents/Style/ImageViewStyle.cs [hunk 3]
```diff
@@ -27,19 +27,12 @@ namespace Tizen.NUI.BaseComponents
...
-
-        static ImageViewStyle() { }
-
         /// This will be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API.
         [EditorBrowsable(EditorBrowsableState.Never)]
         public static readonly BindableProperty ResourceUrlSelectorProperty = BindableProperty.Create("ResourceUrlSelector", typeof(Selector<string>), typeof(ImageViewStyle), null, propertyChanged: (bindable, oldValue, newValue) =>
         {
             var imageViewStyle = (ImageViewStyle)bindable;
+            if (null == imageViewStyle.resourceUrlSelector) imageViewStyle.resourceUrlSelector = new Selector<string>();
             imageViewStyle.resourceUrlSelector.Clone((Selector<string>)newValue);
```
Comment (rabbitfor) [inline src/Tizen.NUI/src/public/BaseComponents/Style/ImageViewStyle.cs]: When the `newValue` is null, next line will cause crash. Is it OK not to test new value's nullity here?

File: src/Tizen.NUI/src/public/BaseComponents/Style/ImageViewStyle.cs [hunk 4]
```diff
@@ -76,6 +69,7 @@ public class ImageViewStyle : ViewStyle
         public static readonly BindableProperty BorderSelectorProperty = BindableProperty.Create("BorderSelector", typeof(Selector<Rectangle>), typeof(ImageViewStyle), null, propertyChanged: (bindable, oldValue, newValue) =>
         {
             var imageViewStyle = (ImageViewStyle)bindable;
+            if (null == imageViewStyle.borderSelector) imageViewStyle.borderSelector = new Selector<Rectangle>();
             imageViewStyle.borderSelector.Clone((Selector<Rectangle>)newValue);
```
Comment (rabbitfor) [inline src/Tizen.NUI/src/public/BaseComponents/Style/ImageViewStyle.cs]: ditto

File: src/Tizen.NUI.Components/Style/SliderStyle.cs (hunk 2 above)
Comment (elishateng) [reply src/Tizen.NUI.Components/Style/SliderStyle.cs]: Ah.. yes.

File: src/Tizen.NUI/src/public/BaseComponents/Style/ImageViewStyle.cs (hunk 3 above)
Comment (elishateng) [reply src/Tizen.NUI/src/public/BaseComponents/Style/ImageViewStyle.cs]: Yes. I'll add null check.

File: src/Tizen.NUI.Components/Style/ButtonStyle.cs (hunk 1 above)
Comment (elishateng) [reply src/Tizen.NUI.Components/Style/ButtonStyle.cs]: Sorry, should use copyfrom.

File: src/Tizen.NUI/src/public/BaseComponents/Style/ImageViewStyle.cs (hunk 3 above)
Comment (elishateng) [reply src/Tizen.NUI/src/public/BaseComponents/Style/ImageViewStyle.cs]: Yes. I'll add null value check

**Review by AdunFang — COMMENTED** [review]
It is OK.

**Pull request merged**